tailwind.config.js

```diff
@@ -31,8 +29,48 @@ const tailwindConfig = {
         },
       },
       colors: {
-        ...colors.light,
         current: 'currentColor',
+        brand: {
```

Comment:
@MeganSteenkamp Great change! Worth considering tailwind's advice going forward - they recommend against using abstract names unless you have a really good reason to

https://tailwindcss.com/docs/customizing-colors#naming-your-colors

Reply:
Thanks @tom-blake. I think next step I'll look at the hex codes and see if they match existing Tailwind classes, if they don't I can refactor to a literal name.